source/api_cc/src/DeepPot.cc

```diff
@@ -175,8 +257,21 @@ DeepPot (const std::string & model, const int & gpu_rank, const std::string & fi
   init(model, gpu_rank, file_content);  
 }
 
+#if HUAWEI_ASCEND
+DeepPot::
+DeepPot (const std::string & model, const int & nloc, const int & npu_rank)
+    : inited (false), init_nbor (false),
+      graph_def(new GraphDef())
+{
+  init(model, nloc, npu_rank);  
+}
+#endif //HUAWEI_ASCEND
+
 DeepPot::~DeepPot() {
   delete graph_def;
+  #if HUAWEI_ASCEND
+  delete session;
+  #endif //HUAWEI_ASCEND
```

Comment:
It looks that `#if` should not be added.

Reply:
Thanks, we have fixed it.

Reply:
I don't see where it is changed.

Reply:
Sorry, now we fix it.